container/BUILD

```diff
@@ -76,77 +76,77 @@ py_binary(
+    ":derivative_with_volume",
+    ":directory_with_tar_base",
+    ":files_base",
+    ":files_with_files_base",
+    ":files_in_layer_with_files_base",
+    ":files_with_tar_base",
+    ":tar_base",
+    ":tar_with_files_base",
+    ":tar_with_tar_base",
+    ":tars_in_layer_with_tar_base",
+    ":docker_tarball_base",
+    ":layers_with_docker_tarball_base",
     # TODO(mattmoor): Re-enable once archive is visible
     # "generated_tarball",
-    "with_unix_epoch_creation_time",
-    "with_millisecond_unix_epoch_creation_time",
-    "with_rfc_3339_creation_time",
-    "with_stamped_creation_time",
-    "with_default_stamped_creation_time",
-    "with_env",
-    "layers_with_env",
-    "with_double_env",
-    "with_label",
-    "with_double_label",
-    "with_stamp_label",
-    "with_user",
-    "workdir_with_tar_base",
-    "link_with_files_base",
-    "build_with_tag",
-    "with_passwd",
-    "with_passwd_tar",
-    "with_group",
-    "with_empty_files",
-    "with_empty_dirs",
-    "gen_image",
-    "data_path_image",
-    "no_data_path_image",
-    "absolute_data_path_image",
-    "root_data_path_image",
-    "dummy_repository",
+    ":with_unix_epoch_creation_time",
+    ":with_millisecond_unix_epoch_creation_time",
+    ":with_rfc_3339_creation_time",
+    ":with_stamped_creation_time",
+    ":with_default_stamped_creation_time",
+    ":with_env",
+    ":layers_with_env",
+    ":with_double_env",
+    ":with_label",
+    ":with_double_label",
+    ":with_stamp_label",
+    ":with_user",
+    ":workdir_with_tar_base",
+    ":link_with_files_base",
+    ":build_with_tag",
+    ":with_passwd",
+    ":with_passwd_tar",
+    ":with_group",
+    ":with_empty_files",
+    ":with_empty_dirs",
+    ":gen_image",
+    ":data_path_image",
+    ":no_data_path_image",
+    ":absolute_data_path_image",
+    ":root_data_path_image",
+    ":dummy_repository",
     # TODO(mattmoor): Re-enable once archive is visible
     # "extras_with_deb",
-    "bundle_test",
-    "stamped_bundle_test",
-    "pause_based",
-    "dashdash_entrypoint",
-    "base_based_warning",
-    "cc_based_warning",
-    "pause_piecemeal",
-    "pause_piecemeal_gz",
-    "cc_image",
+    ":bundle_test",
+    ":stamped_bundle_test",
+    ":pause_based",
+    ":dashdash_entrypoint",
+    ":base_based_warning",
+    ":cc_based_warning",
+    "/pause_piecemeal:image",
+    "/pause_piecemeal_gz:image",
+    ":cc_image",
     # Re-enable once https://github.com/bazelbuild/rules_d/issues/14 is fixed.
     # "d_image",
-    "go_image",
-    "groovy_image",
-    "java_image",
-    "nodejs_image",
-    "py3_image",
-    "py_image",
-    "rust_image",
-    "scala_image",
-    "war_image",
-    "flat",
+    ":go_image",
+    ":groovy_image",
+    ":java_image",
+    ":nodejs_image",
+    ":py3_image",
+    ":py_image",
+    ":rust_image",
+    ":scala_image",
+    ":war_image",
+    ":flat",
 ]
 
 TEST_DATA = [
-    "//testdata:%s.tar" % t
+    "//testdata%s.tar" % t
```

Comment:
I had to change this to not add the ":" since pause_piecemeal is in a subpackage

Reply:
lgtm